website/docs/r/monitor.html.markdown

```diff
@@ -107,7 +107,7 @@ The following arguments are supported:
     Defaults to false.
 * `timeout_h` (Optional) The number of hours of the monitor not reporting data before it will automatically resolve
     from a triggered state. Defaults to false.
-* `include_tags` (Optional) A boolean indicating whether notifications from this monitor will automatically insert its
+* `include_tags` (Optional) A boolean indicating whether notifications from this monitor automatically inserts its triggering tags into the title. Defaults to true.
```

Comment:
"inserts" should actually be "insert", as it refers to plural "notifications", doesn't it?

Reply:
Just updated with a new commit.